python/taichi/main.py

```diff
@@ -191,7 +191,7 @@ def changelog(self, arguments: list = sys.argv[2:]):
             args = parser.parse_args(arguments)
 
             from . import make_changelog
-            res = make_changelog.main(args.version)
+            res = make_changelog.main(args.version, ti.core.get_repo_dir())
```

Comment:
Nice! I think once this is merged in `ti changelog` will be available from anywhere, not just from within root?

Reply:
Yes :)